nautobot_chatops_ipfabric/worker.py

```diff
@@ -107,7 +128,7 @@ def get_user_snapshot(dispatcher):
     context = get_context(dispatcher.context["user_id"])
     snapshot = context.get("snapshot")
     if not snapshot:
-        snapshot = "$last"
+        snapshot = ipfabric_api.get_snapshots()[LAST].snapshot_id
```

Comment:
Can we get a snapshot data that does not contain `LAST`?

Reply:
It will only use last if the user has not set a snapshot.  This is just the default snapshot_id if the user did not perform a set-snapshot

Reply:
Ok, so this is the default snapshot on IP Fabric. My concern was that if a user has not set any snapshot on IP Fabric and they call `get_snapshots`, will `$last` be the snapshot that is returned or will it be empty? I'd like to avoid an indexing error here if there's a scenario where `$last` is not defined.

Reply:
Open new feature to resolve having a snapshot get unloaded which will cause errors for users.
https://github.com/nautobot/nautobot-plugin-chatops-ipfabric/issues/73